src/core/Microsoft.PowerToys.Settings.UI/Controls/BodyTextBlock.xaml.cs

```diff
@@ -82,6 +82,22 @@ public Windows.UI.Text.FontWeight FontWeight
             }
         }
 
+        private FontFamily _fontFamily;
+
+        public FontFamily FontFamily
+        {
+            get
+            {
+                return _fontFamily;
+            }
+
+            set
+            {
+                _fontFamily = value;
```

Comment:
we can use some validation here to make sure the app does not crash on invalid values

Reply:
I quickly checked and I think FontFamily won't crash if wrong/unsupported font is provided. It will just default back to the Apps default font / Segoe UI.

Steps to reproduce:
<TextBlock FontFamily="asdkasdkasdk" /> 

Or would this be something different?